idea/src/org/jetbrains/kotlin/idea/KotlinFoldingBuilder.kt

```diff
@@ -99,10 +100,18 @@ class KotlinFoldingBuilder : CustomFoldingBuilder(), DumbAware {
     override fun getLanguagePlaceholderText(node: ASTNode, range: TextRange): String = when {
         node.elementType == KtTokens.BLOCK_COMMENT -> "/${getFirstLineOfComment(node)}.../"
         node.elementType == KDocTokens.KDOC -> "/**${getFirstLineOfComment(node)}...*/"
+        node.elementType == KtNodeTypes.STRING_TEMPLATE -> "\"\"\"${getFirstLineOfString(node)}...\"\"\""
         node.psi is KtImportList -> "..."
         else ->  "{...}"
     }
 
+    private fun getFirstLineOfString(node: ASTNode): String {
+        val targetStringLine = node.text.split("\n").firstOrNull {
```

Comment:
You can avoid multiple `replace` calls if you use `asSequence().map { it.replace(...) }.firstOrNull { it.isNotBlank }`